orttraining/orttraining/training_api/module.cc

```diff
@@ -68,14 +56,16 @@ Status RemoveUnusedNodes(Graph& inference_graph, InlinedVector<const NodeArg*>&
   return Status::OK();
 }
 
-Status TransformModelOutputsForInference(std::shared_ptr<Model> inference_model, const InlinedVector<std::string_view>& inference_graph_outputs) {
+Status TransformModelOutputsForInference(Graph& inference_graph,
+                                         const InlinedVector<std::string_view>& inference_graph_outputs) {
+  // Model is updated to remove any outputs that are not defined in inference_graph_outputs. Nodes
+  // producing these unused model outputs are also subsequently removed.
+
   if (inference_graph_outputs.empty()) {
     // The outputs are not decipherable, so don't transform the graph outputs.
```

Comment:
I feel this somehow hide the error when user pass an empty graph output by mistake? shall we make it an error?